proxy/src/main/java/com/velocitypowered/proxy/network/ConnectionManager.java

```diff
@@ -123,8 +122,8 @@ public void bind(final InetSocketAddress address) {
         .childOption(ChannelOption.IP_TOS, 0x18)
         .localAddress(address);
 
-    if (transportType == TransportType.EPOLL && server.getConfiguration().useTcpFastOpen()) {
-      bootstrap.option(EpollChannelOption.TCP_FASTOPEN, 3);
+    if (server.getConfiguration().useTcpFastOpen()) {
+      bootstrap.option(ChannelOption.TCP_FASTOPEN, 3);
```

Comment:
This option is only supported on epoll & io_uring, on all other channel types netty will show a warning when setting this option. I'm not sure if this is a problem, since the user has to explicitly enable the option in the configuration file anyway. (same applies for `TCP_FASTOPEN_CONNECT` below)

Reply:
I'm not even sure tcp_fastopen is relevant given that the client and the server don't even enable it last I knew

Reply:
@derklaro In my test, there is no warning when using NIO transport.